fedrec/modules/embeddings.py

```diff
@@ -24,13 +24,19 @@ def md_solver(n, alpha, d0=None, B=None, round_dim=True, k=None):
     '''
     An external facing function call for mixed-dimension assignment
     with the alpha power temperature heuristic
-    Inputs:
+    
+    Arguments
+    ----------
     n -- (torch.LongTensor) ; Vector of num of rows for each embedding matrix
     alpha -- (torch.FloatTensor); Scalar, non-negative, controls dim. skew
     d0 -- (torch.FloatTensor); Scalar, baseline embedding dimension
     B -- (torch.FloatTensor); Scalar, parameter budget for embedding layer
     round_dim -- (bool); flag for rounding dims to nearest pow of 2
     k -- (torch.LongTensor) ; Vector of average number of queries per inference
```

Comment:
It will look good.  If you can write as ``` n(torch.LongTensor) - ```  instead of  ```n -- (torch.LongTensor);```  and make the proper indentation as well.
```suggestion
        n -- (torch.LongTensor) ; Vector of num of rows for each embedding matrix
        alpha -- (torch.FloatTensor); Scalar, non-negative, controls dim. skew
        d0 -- (torch.FloatTensor); Scalar, baseline embedding dimension
        B -- (torch.FloatTensor); Scalar, parameter budget for embedding layer
        round_dim -- (bool); flag for rounding dims to nearest pow of 2
        k -- (torch.LongTensor) ; Vector of average number of queries per inference
```